learn/configuration/instance_options.md

```diff
@@ -169,16 +171,53 @@ Sets the directory where Meilisearch will create dump files.
 
 ### Import dump
 
+::: warning
+This is a CLI flag and does not take any values. Assigning a value will throw an error.
+:::
+
```

Comment:
This is not a CLI flag, it requires a filepath pointing to a `.dump` file. Suggest deleting this.

```suggestion
```